### DISCUSSION TITLE
Inserting new proteases into DIA-NN

### DISCUSSION THREAD

**Discussion by dperezhlih** [vdemichev/DiaNN#532] [Q&A]
Dear Mr Demichev,

Thanks very much for your nice software, it is very straightforward and easy to use, also very nice results coming from it. I am currently working with other alternative proteases for Mass Spectrometry, using the TIMS-TOF for the analysis. I would like to do DIA on my samples but I don't know how to setup different enzymes without peptide library (library free for DIA-PASEF). Is there any way I can use DIA-NN inserting other proteases for my search? Thank you very much for your attention.
Best,

Daniel

**Comment by vdemichev** [comment]
Hi Daniel,

Please see the --cut command description in the docs.

Best,
Vadim

**Comment by dperezhlih** [comment]
Thank you very much!
